CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## UNRELEASED
 
 ### Added
+
+- `quint run` produces a friendlier message when it meets a `const` (#1050)
```

Comment:
Should this be under `Changed`?

Reply:
Yeah, I also thought about this section.

Reply:
I’d say fixed :)

changed is meant to signal breaking changes I think, and add is for new features, but not-great error messages seem more like bugs to me. See https://keepachangelog.com/en/1.0.0/ for reference.

Reply:
Oh, right, I didn't see the `Fixed` section below.
Should go there.